<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="presetsCombo">
     <item>
      <property name="text">
       <string>[presets]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 2 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 2 3 8 9 10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 2 3 4 5 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 2 3 12</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>interpret the values in each line as:</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>282</width>
        <height>235</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Separator:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioSpace">
        <property name="text">
         <string>space</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioTab">
        <property name="text">
         <string>tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSemi">
        <property name="text">
         <string>;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioComma">
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Skip first</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="suffix">
        <string> lines</string>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSpace</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>onSepRadio(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSemi</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>onSepRadio(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioComma</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>onSepRadio(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioTab</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>onSepRadio(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>onLineNb(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>presetsCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>onPreset(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSepRadio(bool)</slot>
  <slot>onLineNb(int)</slot>
  <slot>onPreset(QString)</slot>
 </slots>
</ui>
